src/test/java/games/strategy/engine/framework/map/download/DownloadUtilsTests.java

```diff
@@ -105,22 +102,18 @@ private void downloadToFile() throws Exception {
     public void shouldThrowExceptionWhenStatusCodeIsNotOk() {
       when(statusLine.getStatusCode()).thenReturn(HttpStatus.SC_NOT_FOUND);
 
-      catchException(() -> downloadToFile());
+      final IOException e = assertThrows(IOException.class, () -> downloadToFile());
```

Comment:
Half way through replacing I noticed I could inline the assertThrows statement...
I did that for the remaining replacements, should we be consistent with this?

Reply:
When I first read your comment, I thought there was a way to chain the assertions similar to how Hamcrest does it.  Instead, it seems we're embedding one assertion inside of another.  That is a bit hard to read because it further convolutes the act/when portion of the test with the assert/then portion.

IMO, in the cases where we need to perform additional assertions on the exception, we should just assign the return value of `assertThrows()` to an intermediate variable, and then use the intermediate variable in subsequent assertions.  Note that it's probably not necessary to use the exact exception type for the intermediate variable.  We can probably get away with simply using `Throwable` or `Exception` if it makes it easier to write.